tests/integration/source_code/test_jobs.py

```diff
@@ -48,15 +45,18 @@ def test_running_real_workflow_linter_job(installation_ctx, make_notebook, make_
     if result['count'] == 0:
         installation_ctx.deployed_workflows.relay_logs("experimental-workflow-linter")
         assert False, "No workflow problems found"
+    dfsa_records = installation_ctx.directfs_access_crawler_for_paths.snapshot()
+    assert dfsa_records
```

Comment:
just some record? Or a specific amount?

Reply:
```suggestion
    assert dfsa_records == [...]
```
can we please assert for specific values?